cli_reference/basic_cli_operations.adoc

```diff
@@ -117,6 +118,7 @@ syntax:
 |`pod` |`po`
 |`ResourceQuota` | `quota`
 |`replicationController` |`rc`
+|`replicaSet` (Technology Preview)|`rs`
```

Comment:
@kargakis Correct to say that replica sets are also Tech Preview, if deployments are?